[2023-05-05T11:37:22.327+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-05-04T00:00:00+00:00 [queued]>
[2023-05-05T11:37:22.333+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-05-04T00:00:00+00:00 [queued]>
[2023-05-05T11:37:22.334+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-05-05T11:37:22.344+0000] {taskinstance.py:1350} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-05-04 00:00:00+00:00
[2023-05-05T11:37:22.347+0000] {standard_task_runner.py:57} INFO - Started process 1185 to run task
[2023-05-05T11:37:22.350+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2023-05-04T00:00:00+00:00', '--job-id', '1129', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp_172xh4q']
[2023-05-05T11:37:22.352+0000] {standard_task_runner.py:85} INFO - Job 1129: Subtask insert_into_table
[2023-05-05T11:37:22.409+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2023-05-04T00:00:00+00:00 [running]> on host 39440b36e721
[2023-05-05T11:37:22.507+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jose-holgado' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-05-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-04T00:00:00+00:00'
[2023-05-05T11:37:22.509+0000] {sql.py:262} INFO - Executing: 
            INSERT INTO dag_runs (dt, dag_id) VALUES ('2023-05-04', 'dag_with_postgres_operator_v03')
        
[2023-05-05T11:37:22.526+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-05T11:37:22.587+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-05T11:37:22.601+0000] {sql.py:375} INFO - Running statement: 
            INSERT INTO dag_runs (dt, dag_id) VALUES ('2023-05-04', 'dag_with_postgres_operator_v03')
        , parameters: None
[2023-05-05T11:37:22.606+0000] {sql.py:384} INFO - Rows affected: 1
[2023-05-05T11:37:22.621+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20230504T000000, start_date=20230505T113722, end_date=20230505T113722
[2023-05-05T11:37:22.685+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-05T11:37:22.713+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
